auth-web/src/views/auth/PriceListView.vue

```diff
@@ -0,0 +1,219 @@
+<template>
+  <main class="view-container container">
+      <h1 class="mb-4">Price List</h1>
+      <p class="mb-8">Where applicable, tax is included in the price shown. Prices are subject to change without notice.</p>
+      <section class="mb-10">
+        <h2 class="mb-5">Business Registry</h2>
+        <v-card flat>
+          <table aria-label="Price List Table">
+            <thead>
+              <th
+                class="product text-left"
+                scope="col"
+              >
+                Product
+              </th>
+              <th
+                class="fee"
+                scope="col"
+              >
+                Price
+              </th>
+              <th
+                class="fee--service"
+                scope="col"
+              >
+                Service Fee
+              </th>
+              <th
+                class="fee--priority"
+                scope="col"
+              >
+                Priority Fee
+              </th>
+              <th
+                class="fee--future"
+                scope="col"
+              >
+                Future Effective Fee
+              </th>
+            </thead>
+            <tbody>
+              <tr v-for="item in pricelist" :key="item.name">
+                <td
+                  class="product font-weight-bold"
+                >
+                  {{item.name}}
+                </td>
+                <td
+                  class="fee">
+                  <span aria-describedby="Annual Report PRice">{{'$' + item.price}}</span>
+                </td>
+                <td
+                  class="fee--service"
+                >
+                  {{'$' + item.serviceFee}}
+                </td>
+                <td
+                  class="fee--priority"
+                >
+                  <span v-if="item.priorityFee">{{'$' + item.priorityFee}}</span>
+                </td>
+                <td
+                  class="fee--future"
+                >
+                  <span v-if="item.futureEffectiveFee">{{'$' + item.futureEffectiveFee}}</span>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </v-card>
+      </section>
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import { VueConstructor } from 'vue'
```

Comment:
Looks like this is not needed. we are not using it anywhere, can you crosscheck?